src/confluent_kafka/src/Consumer.c

```diff
@@ -982,27 +982,22 @@ static PyObject *Consumer_poll (Handle *self, PyObject *args,
 
 static PyObject *Consumer_memberid (Handle *self, PyObject *args,
                                     PyObject *kwargs) {
-        char *memberid = NULL;
+        char *memberid;
         PyObject *memberidobj;
-        CallState cs;
         if (!self->rk) {
                 PyErr_SetString(PyExc_RuntimeError,
                                 "Consumer closed");
                 return NULL;
         }
 
-        CallState_begin(self, &cs);
-
         memberid = rd_kafka_memberid(self->rk);
 
-        if (!CallState_end(self, &cs)) {
+        if (!memberid)
                 return NULL;
```

Comment:
Returning null requires an exception to have been raised, but in this case we just want to return None.